D1673/P1673.md

```diff
@@ -7318,11 +7315,12 @@ sum_of_squares_result<T> vector_sum_of_squares(
 <i>[Note:</i> These functions correspond to the LAPACK function `xLASSQ`.
 <i>-- end note]</i>
 
-[1]{.pnum} *Preconditions:*
+[1]{.pnum} *Mandates:*
 
-  * [1.1]{.pnum} `T` shall be *Cpp17MoveConstructible* and *Cpp17LessThanComparable*, and
+  * [1.1]{.pnum} `T` meets both the *Cpp17MoveConstructible* and
```

Comment:
1. Can this first point actually be a Mandate, given that the semantic parts of the named requirements aren't checkable at compile time?

2. Now that we have `std::move_constructible`, should we consider using that instead?  (LessThanComparable is a strict weak ordering, which doesn't have a named C++20 concept.)

Reply:
1) Yes; this is done elsewhere in the standard already. 2) It depends on the exact semantics that you want, and I don't know the details well enough to answer that.  It might help to try it both ways and write some simple compile tests with different template parameters and see what makes the most sense.